src/nurse_ussd.js

```diff
@@ -258,6 +264,90 @@ go.app = function() {
             });
         });
 
+        self.add('st_change_id_no', function(name) {
+            var owner = self.user.extra.nc_working_on === "" ? 'your' : 'their';
+            return new ChoiceState(name, {
+                question: $("Please select {{owner}} type of identification:")
+                    .context({owner: owner}),
+                choices: [
+                    new Choice('st_id_no', $('RSA ID')),
+                    new Choice('st_passport', $('Passport'))
+                ],
+                next: function(choice) {
+                    return choice.value;
+                }
+            });
+        });
+
+        self.add('st_id_no', function(name) {
+            var owner = self.user.extra.nc_working_on === "" ? 'your' : 'their';
+            var error = $("Sorry, the format of the ID number is not correct. Please enter it again, e.g. 11118888:");
+            return new FreeText(name,{
+                question: $("Please enter {{owner}} 13-digit ID number:")
+                    .context({owner:owner}),
+                check: function(content) {
+                    if(!go.utils.validate_id_sa(content)){
+                        return error;
+                    }else{
+                        return null;
+                    }
+                },
+                next: function(content) {
+                    return 'end_change_id';
+                }
+            });
+        });
+
+        self.add('st_passport', function(name) {
+            return new ChoiceState(name, {
+                question: $('What is the country of origin of the passport'),
+                choices: [
+                    new Choice('na', $('Namibia')),
+                    new Choice('bw', $('Botswana')),
+                    new Choice('mz', $('Mozambique')),
+                    new Choice('sz', $('Swaziland')),
+                    new Choice('ls', $('Lesotho')),
+                    new Choice('cu', $('Cuba')),
+                    new Choice('other', $('Other')),
+                ],
+                next: 'st_passport_no'
+            });
+        });
+
+        self.add('st_passport_no', function(name) {
+            return new FreeText(name, {
```

Comment:
What happened to the `var error` and `var question` parts?  I suggest copying the whole `self.add` code block again from the original and just updating the state names.